_posts/2017-06-05-slack-prometheus-alertmanager.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: Slack Notifications & Alertmanager 
+author: ed_marshall
+image: phone.jpg
+---
+
+
+A simple guide, taking a look at how we can enhance the quality of the alert notifications we receive from Alertmanager in our Slack channels. The changes we run through allow us to enrich the quality of the information in our alerts, allowing us to make better decisions up-front when responding to an event.
+
+Alertmanager is the go-to method of alert handling, routing and delivery for Prometheus. It's a fantastic yet simple tool that allows us to customise how we handle the alerting aspects of our monitoring systems. The process of installing and configuring Alertmanager is already well [documented](https://www.robustperception.io/using-slack-with-the-alertmanager/).
+
+Based on the same powerful PromQL that Prometheus uses, it's easy to see why it's a popular choice. This article hopes to offer a simple guide to getting more value out of Alertmanager when used to drive notifications, in our case into Slack.
+
+# But Why?
```

Comment:
These sub-titles are a bit vague. If they were more specific then it will help people coming back to this blog as a reference. This one could be "What's wrong with the default notifications?". The next one, rather than "Another way", could be "Adding more context".